lib/base-controller.js

```diff
@@ -1,9 +1,24 @@
 'use strict';
 
-var Controller = require('hmpo-form-wizard').Controller;
+var Controller = require('hmpo-date-controller');
 var util = require('util');
 var _ = require('lodash');
 
+// In HOF we are more lenient with date inputs than in vanilla
+// hmpo-form-wizard and so must override some validators
+var validators = Controller.validators;
+validators = _.extend(validators, {
+  'date-year': function hofdateyear(value) {
```

Comment:
I think the key should be hypenated as is - this pattern is used throughout, with hmpo template mixins etc. Unless you meant the function name -> `hofDateYear` ?